src/main/java/com/redhat/red/build/koji/model/xmlrpc/messages/GetTaskRequest.java

```diff
@@ -28,14 +28,29 @@
     @DataIndex( 0 )
     private int taskId;
 
+    @DataIndex( 1 )
+    private boolean request;
+
```

Comment:
After a quick look at the api doc of getTaskInfo, the default value of the param "request" is false, so this change should make sense.